ui/components/SearchBar/index.js

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect, useRef } from "react";
+import { SearchIcon, CloseIcon } from "../Icons";
+
```

Comment:
We want to slowly move to ts for all files so better to have that by default when creating a new file. If you run into any annoying type issues it's fine to use `any` though